zerver/lib/actions.py

```diff
@@ -6260,11 +6260,15 @@ def filter_presence_idle_user_ids(user_ids: Set[int]) -> List[int]:
     return sorted(idle_user_ids)
 
 
-def do_send_confirmation_email(invitee: PreregistrationUser, referrer: UserProfile) -> str:
+def do_send_confirmation_email(
+    invitee: PreregistrationUser, referrer: UserProfile, invite_expires_in_days: int
```

Comment:
Can the new parameter be an optional kwarg?  I imagine in most cases, we'll be using the defaults.